reference/carts.v3.yml

```diff
@@ -1711,13 +1709,11 @@ components:
                     type: number
                     description: The product option value identifier in number format.
                     example: 128
-                title: Product Option
           required:
             - variant_id
             - product_id
             - quantity
-        - type: object
```

Comment:
`type: object` is required and should not be removed.

Reply:
@kzhang-dsg I see `type: object` on line 1697

Reply:
For some reason I dont see type: object on line 1697. I check ed both the file in this commit and the file in the main branch.

See the file in this commit: https://github.com/bigcommerce/api-specs/blob/e0d7b063ea77955a71b590ad09357cbff75adf8a/reference/carts.v3.yml
![image](https://user-images.githubusercontent.com/15915824/229573061-c7a60243-e20e-4a3b-bb67-8ed382ce8f10.png)